### DISCUSSION TITLE
Do we have to be on the same wifi as the person to be connected?

### DISCUSSION THREAD

**Discussion by oguzkaan78** [rustdesk/rustdesk#2901] [General]
When I try with my device to my other device, it runs perfectly but when I try to with someone, it gives error.

**Comment by SsuJojo** [comment]
“Do we have to be on the same wifi as the person to be connected? ”
-----------------------
 The answer is no. 
 But if you're on the same wifi, rustdesk will use p2p for better speed, security and stability.

**Comment by SsuJojo** [comment]
the run error may be due to a remote server that cannot be connecte
